README.md

```diff
@@ -315,7 +315,9 @@ This is not very common, and if you are considering setting it up, you probably
 
 As with encryption software, we do not have specific requirements for what kind of solution you will use.
 The only requirement is the license.
-We recommend [VirtualBox](https://www.virtualbox.org/).
+- We generally recommend [VirtualBox](https://www.virtualbox.org/).
+
```

Comment:
we are introducing a linter which would be offended by that extra empty line - please remove it
```suggestion
```